MarkBind/markbind
# Fix Footer Issue On Medium Viewports

**What is the purpose of this pull request?**




Fixes #1634 

**Overview of changes:**
Remove the margin-left of the footer in the default CSS template.

**Anything you'd like to highlight / discuss:**

**Testing instructions:**
Run `markdown init` and resize the viewport; there should be no left margin for the footer at any point.



**Proposed commit message: (wrap lines at 72 characters)**

```
Fix footer margin on medium viewports
```


---

**Checklist:** :ballot_box_with_check:
